doc/user_guide/install.rst

```diff
@@ -51,6 +51,31 @@ Quick instructions to install HyperSpy using Anaconda (Linux, MacOs, Windows)
        $ pip install hyperspy
 
 .. warning::
+
+    HyperSpy requires two packages, `traits` and `traitsui` which are not
+    written in pure Python and, therefore, needs to be compiled. The above
+    command will download these packages and try to compile them automatically.
+    However, if the required development tools are not installed in your system
+    the installation will fail. This is more likely to be the case in Microsoft
+    Windows and (to a lesser extend) MacOS. Microsoft Windows users could
+    install the HyperSpy Bundle instead which includes all the required
+    libraries. Alternatively, they could download and install `traits` and
+    `traitsui` from `Christoph Gohlke's repository
+    <http://www.lfd.uci.edu/~gohlke/pythonlibs/>`_.
+
+    As of the 27th of May 2016, Anaconda does not distribute `traits` and
+    `traitsui` for Python 3. Once it does, the simples solution is to install
+    those packages using `conda` before installing HyperSpy as above e.g.:
```

Comment:
The section above is a little unclear... You mean that once Anaconda stops distributing the packages you should install using conda?